app/views/includes/foot.jade

```diff
@@ -44,5 +45,20 @@ script(type='text/javascript', src='/js/init.js')
 //Socket.io Client Library
 script(type='text/javascript', src='/socket.io/socket.io.js')
 
+// firebase
+script(type='text/javascript', src='https://www.gstatic.com/firebasejs/3.5.0/firebase.js')
+script(type='text/javascript', src='https://www.gstatic.com/firebasejs/3.5.0/firebase-app.js')
+script(type='text/javascript', src='https://www.gstatic.com/firebasejs/3.5.0/firebase-database.js')
+script(type='text/javascript', src='https://www.gstatic.com/firebasejs/3.5.0/firebase-messaging.js')
+script.
+  var config = {
```

Comment:
Not a good practise to have the config files here. Should be stored somewhere in a config service.